app/views/training_modules/show.html.slim

```diff
@@ -1,50 +1,26 @@
 - content_for(:head) do 
   = tag.meta name: 'turbo-visit-control', content: 'reload'
 
-= govuk_back_link href: my_modules_path
+= govuk_back_link text: 'Back to My modules', href: my_modules_path
```

Comment:
We have a `back_to_module_overview_button` method which should be moved to the LinkHelper for consistency, can you do that in this branch please.

Reply:
Will do. I actually think we need a back button overhaul - they're supposed to be outside <main> and they aren't always. It's been in the back of my mind for a while